build_mac.sh

```
@@ -2,6 +2,8 @@
 
 echo "Building cryptomator cli..."
 
+APP_VERSION='0.1.0-local'
```

Shellcheck (0.9.0-1)

[warning] APP_VERSION appears unused. Verify use (or export if used externally).

(SC2034)